READMEs/c2d-flow.md

```diff
@@ -90,26 +72,20 @@ DATASET_compute_values = {
 }
 
 # Create the Service
-from ocean_lib.services.service import Service
```

Comment:
Don't delete this line - you still need access to the `Service` class below.

Reply:
Done